Add latest and missing changes to game fixes document (#448)
xezon/GeneralsGamePatch: Patch104pZH/Design/Changes/game_fixes.md

| C = Controversial |
| A = Accepted      |

| C | A | ID                                                          | Change                                                                                                                |
|---|---|-------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------|
|   |   | [#2](https://github.com/xezon/GeneralsGamePatch/pull/2)     | Fixed issue that showed wrong model of GLA Battle Bus in bunkered state                                               |
|   |   | [#3](https://github.com/xezon/GeneralsGamePatch/pull/3)     | Fixed exploit that granted USA a significant amount of cash after building a Drop Zone                                |

> |   |   | [#7](https://github.com/xezon/GeneralsGamePatch/pull/7)     | Fixed issue that would permanently reduce the power level of a faction                                                |

Reviewer: ```suggestion
|   |   | [#7](https://github.com/xezon/GeneralsGamePatch/pull/7)     | Fixed issue that would permanently reduce the power level of a player                                                |
```

> |   |   |                                                             | Improved the performance of USA Patriot missile assist effect by around 60%                                           |

Reviewer: ```suggestion
|   |   |                                                             | Reduced lag of USA Patriot missile assist effect by around 60%                                           |
```

| C |   |                                                             | Removed Heat Haze effect from Microwave Tank and GPS Scrambler to avoid critical rendering glitches                   |
|   |   |                                                             | Fixed critical issue that crashed all clients in a match                                                              |
|   |   |                                                             | Fixed issue that made units keep firing onto already killed infantry units                                            |
|   |   |                                                             | Fixed issue that spawned wrong USA Ranger types on Particle Cannon kills                                              |
|   |   |                                                             | Fixed issue that prevented a stealthed and attacking GLA Palace to reveal itself                                      |
|   |   | [#75](https://github.com/xezon/GeneralsGamePatch/pull/75)   | Fixed issue where a player would not be defeated by just owning a GLA Tunnel/Stinger hole                             |

> |   |   | [#76](https://github.com/xezon/GeneralsGamePatch/pull/76)   | Added red bullet tracers to heroic USA Pathfinders and GLA JarmenKell                                                 |

Reviewer: ```suggestion
|   |   | [#76](https://github.com/xezon/GeneralsGamePatch/pull/76)   | Added red bullet tracers to heroic USA Pathfinders and GLA Jarmen Kell                                                 |
```

|   |   | [#78](https://github.com/xezon/GeneralsGamePatch/pull/78)   | Fixed wrong Anthrax effect colors on Toxin Tractor puddles                                                            |
|   |   | [#94](https://github.com/xezon/GeneralsGamePatch/pull/94)   | Fixed issue that allowed to trigger demo traps by placing scaffolds nearby                                            |
|   |   | [#85](https://github.com/xezon/GeneralsGamePatch/pull/85)   | Fixed issue that caused Toxin demo trap to prematurely create toxin puddle before explosion                           |
|   |   | [#95](https://github.com/xezon/GeneralsGamePatch/pull/95)   | Fixed GLA Stealth Saboteur death voice                                                                                |
|   |   | [#96](https://github.com/xezon/GeneralsGamePatch/pull/96)   | Fixed issue that sent the China Supply Truck flying high in the sky when shot                                         |
| C |   | [#97](https://github.com/xezon/GeneralsGamePatch/pull/97)   | Fixed issue that made it difficult to drive over the China Infantry Minigunner                                        |

> |   |   | [#83](https://github.com/xezon/GeneralsGamePatch/pull/83)   | Fixed issue that had Toxin Shells lose Projectile effect after acquiring Anthrax Beta                                 |

Reviewer: ```suggestion
|   |   | [#83](https://github.com/xezon/GeneralsGamePatch/pull/83)   | Fixed issue that had Toxin Shells lose projectile effect after acquiring Anthrax Beta                                 |
```

|   |   | [#84](https://github.com/xezon/GeneralsGamePatch/pull/84)   | Fixed issue that made Gamma Toxin Streams have green particles when clearing buildings                                |
|   |   | [#86](https://github.com/xezon/GeneralsGamePatch/pull/86)   | Fixed wrong UI portrait image of GLA Sneak attack                                                                     |
|   |   | [#91](https://github.com/xezon/GeneralsGamePatch/pull/91)   | Fixed wrong UI ability image of GLA Saboteur                                                                          |

> |   |   | [#87](https://github.com/xezon/GeneralsGamePatch/pull/87)   | Fixed issue that made Demo GLA Rocket Buggy lose red missile effect after acquiring Rocket Upgrade                    |

Reviewer: ```suggestion
|   |   | [#87](https://github.com/xezon/GeneralsGamePatch/pull/87)   | Fixed issue that made Demo GLA Rocket Buggy lose red missile effect after acquiring Buggy Ammo Upgrade                    |
```
Otherwise could be confused with AP Rockets.

|   |   | [#231](https://github.com/xezon/GeneralsGamePatch/pull/231) | Fixed misaligned propaganda effect on China Assault Troopcrawler                                                      |
|   |   | [#231](https://github.com/xezon/GeneralsGamePatch/pull/231) | Fixed misaligned propaganda effect on China Attack Outpost                                                            |
|   |   | [#207](https://github.com/xezon/GeneralsGamePatch/pull/207) | Added missing AP Rocket Upgrade icon to GLA SCUD Launcher cameo                                                       |
|   |   | [#224](https://github.com/xezon/GeneralsGamePatch/pull/224) | Added missing Patriotism Upgrade icon to Infantry China Tank Hunter                                                   |
|   |   | [#225](https://github.com/xezon/GeneralsGamePatch/pull/225) | Added missing Upgrade icons to GLA Rebels                                                                             |
|   |   | [#205](https://github.com/xezon/GeneralsGamePatch/pull/205) | Fixed wrong upgrade icon placements on USA Tomahawk                                                                   |

> |   |   | [#209](https://github.com/xezon/GeneralsGamePatch/pull/209) | Fixed China Nuklear Missile exploit                                                                                   |

Reviewer: ```suggestion
|   |   | [#209](https://github.com/xezon/GeneralsGamePatch/pull/209) | Fixed China Nuclear Missile exploit                                                                                   |
```
a.k.a. nucular
